distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Named_Pattern.enso

```diff
@@ -44,6 +47,7 @@ type Named_Pattern
         Named_Pattern.Trailing_Whitespace -> "\s+$"
         Named_Pattern.Duplicate_Whitespace -> "(?<=\s)\s+"
         Named_Pattern.All_Whitespace -> "\s+"
+        Named_Pattern.Newlines -> "(\r\n?|\n)"
```

Comment:
Shouldn't this be a non-capturing group?

Reply:
Yes good catch